<launch>
  <push-ros-namespace namespace="traffic_light_arbiter"/>
  <node_container pkg="rclcpp_components" exec="component_container" name="container" namespace="">
    <composable_node pkg="traffic_light_arbiter" plugin="TrafficLightArbiter" name="converter">
      <remap from="~/sub/traffic_lights" to="/perception/traffic_light_recognition/traffic_signals"/>
      <remap from="~/pub/traffic_lights" to="/perception/traffic_light_selector/internal/traffic_lights"/>
    </composable_node>
  </node_container>
</launch>
